packages/components/NewTeamModal/renderTeamAvatars.js

```diff
@@ -139,11 +139,21 @@ export const renderTeamAvatars = ({ icon }, setFieldValue) => {
   const shouldShowUploader =
     (icon && IMAGE_BASE64_LIST.includes(icon.raw)) || !icon;
 
-  const showErrors = rejectedFiles => {
+  const showErrors = rejectedFiles => {console.log(rejectedFiles);
```

Comment:
pls, do not use the console when possible.

Reply:
Removed